src/dymaptic.GeoBlazor.Core/Scripts/jsBuilder.ts

```diff
@@ -1163,6 +1164,13 @@ export async function buildJsPopup(dotNetPopup: any, viewId: string): Promise<Po
         spinnerEnabled: dotNetPopup.spinnerEnabled ?? true
     });
     
+    if (hasValue(dotNetPopup.autoOpenEnabled)) {
```

Comment:
if `autoOpenEnabled` is listed as obsolete, do we want to update the `autoOpenEnabled` references and replace all references with popupEnabled?

Reply:
In the samples? Yes, we should do that. I thought I had, but I will double check.